runtime_scan/pkg/orchestrator/config.go

```diff
@@ -123,18 +118,25 @@ func setConfigDefaults(backendHost string, backendPort int, backendBaseURL strin
 	viper.SetDefault(ScanResultProcessorReconcileTimeout, scanresultprocessor.DefaultReconcileTimeout.String())
 	viper.SetDefault(DiscoveryInterval, discovery.DefaultInterval.String())
 	viper.SetDefault(ControllerStartupDelay, DefaultControllerStartupDelay.String())
+	viper.SetDefault(ProviderKind, DefaultProviderKind)
 
 	viper.AutomaticEnv()
 }
 
 func LoadConfig(backendHost string, backendPort int, baseURL string) (*Config, error) {
 	setConfigDefaults(backendHost, backendPort, baseURL)
 
-	c := &Config{
-		AWSConfig: aws.LoadConfig(),
+	var providerKind models.CloudProvider
+	switch strings.ToLower(viper.GetString(ProviderKind)) {
+	case "aws":
```

Comment:
why not to use `strings.ToLower(models.AWS)`?

Reply:
Fixed.